### PR TITLE
Add Charlot Shaw's paper

### PR DESCRIPTION
**Pull request by UnwarySage**
I'm working on improving my resume, and would like an official spot to link to when I discuss my seminar.
I don't have the Ruby toolchain installed, so can't verify that this builds, but I believe it should.
Thanks in advance,
Charlot

### CODE REVIEW

**Review by floogulinc — CHANGES_REQUESTED** [review]
I would ask you to please use the same naming convention we use for the other seminars. Your paper should be `shaw.pdf` and your slides `shawslides.pdf`

**Comment by UnwarySage** [comment]
Have done so

File: _seminars/spring2020/index.md [hunk 1]
```diff
@@ -73,6 +73,8 @@ papers:
     author: Charlot Shaw
     time: 2020-04-18 5:00 PM
     room: Zoom 2
+    pdf: ShawSeminar.pdf
+    slides: ShawSlides.pdf
```
Comment (floogulinc) [inline _seminars/spring2020/index.md:77]: You need to update the list too.

```suggestion
    pdf: shaw.pdf
    slides: shawslides.pdf
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Comment by floogulinc** [comment]
Oh also, even if you don't have all the Ruby stuff installed you can easily build and preview the site locally with Docker. This is from the README:

<blockquote>

### Running with Docker

If you have Docker you can run the site with docker using

```bash
docker run -it --rm -v ${PWD}:/usr/src/app -p 4000:4000 starefossen/github-pages
```

This will serve the site on port 4000 (it should also automatically rebuild the site when changes are made to the files).

</blockquote>

**Comment by UnwarySage** [comment]
Realized the `index.md` needed to be updated as well, unfortunately after pushing.

File: _seminars/spring2020/index.md [hunk 2]
```diff
@@ -73,6 +73,8 @@ papers:
     author: Charlot Shaw
     time: 2020-04-18 5:00 PM
     room: Zoom 2
+    pdf: shawseminar.pdf
```
Comment (floogulinc) [inline _seminars/spring2020/index.md:76]: This is still incorrect.

```suggestion
    pdf: shaw.pdf
```

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Comment by UnwarySage** [comment]
I believe I've settled the outstanding issues.
Do you see anything else on your end?

**Pull request merged**